payments_service/braintree/views.py

```diff
@@ -80,3 +81,36 @@ def set_up_customer(self, buyer_uuid):
             log.info('creating new braintree customer for {buyer}'
                      .format(buyer=buyer_uuid))
             self.api.braintree.customer.post({'uuid': buyer_uuid})
+
+
+class PlainTextRenderer(BaseRenderer):
+    media_type = 'text/plain'
+    format = 'txt'
+
+    def render(self, data, media_type=None, renderer_context=None):
+        return data.encode(self.charset)
+
+
+class Webhook(UnprotectedAPIView, SolitudeBodyguard):
+    methods = ['post', 'get']
+
+    def perform_content_negotiation(self, request, **kw):
+        # Braintree sends a request that has an Accept Header of
+        # u'*/*; q=0.5', u'application/xml', but we need to force
+        # DRF to return it as text/plain. The only way we can do that
+        # is to override the content negotiation and insert our rather
+        # boring plain text renderer.
+        if request.method.lower() == 'get':
+            renderer = PlainTextRenderer()
+            return (renderer, renderer.media_type)
+        return (super(self.__class__, self)
+                .perform_content_negotiation(request, **kw))
+
+    def get(self, request, **kw):
+        self.resource = 'braintree.webhook.verify'
+        return super(self.__class__, self).get(
+            request, **dict(request.QUERY_PARAMS.items()))
```

Comment:
I wonder if it would be better just to refactor ``SolitudeBodyguard`` so you could do this:
````python
class Webhook(...):
    resource = {'get': 'braintree.webhook.verify', 'post': 'braintree.webhook.parse'}
````
````